[2023-01-04 04:09:36,837] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-24T07:00:00+00:00 [queued]>
[2023-01-04 04:09:36,847] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-24T07:00:00+00:00 [queued]>
[2023-01-04 04:09:36,847] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:09:36,847] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 04:09:36,847] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:09:36,876] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-02-24 07:00:00+00:00
[2023-01-04 04:09:36,881] {standard_task_runner.py:52} INFO - Started process 61485 to run task
[2023-01-04 04:09:36,886] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-02-24T07:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmpg2pf77ol', '--error-file', '/tmp/tmp2gdt19rs']
[2023-01-04 04:09:36,887] {standard_task_runner.py:80} INFO - Job 83: Subtask get_crypto_values
[2023-01-04 04:09:36,964] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-24T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 04:09:37,043] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-02-24T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-24T07:00:00+00:00
[2023-01-04 04:09:38,005] {logging_mixin.py:115} INFO - Encountered errors while inserting rows: [{'index': 0, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '25-02-2021'"}]}, {'index': 1, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '25-02-2021'"}]}]
[2023-01-04 04:09:38,005] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 04:09:38,013] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210224T070000, start_date=20230104T070936, end_date=20230104T070938
[2023-01-04 04:09:38,090] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 04:09:38,128] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-04 05:24:17,592] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-24T07:00:00+00:00 [queued]>
[2023-01-04 05:24:17,654] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-24T07:00:00+00:00 [queued]>
[2023-01-04 05:24:17,654] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:24:17,654] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 05:24:17,655] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:24:17,667] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-02-24 07:00:00+00:00
[2023-01-04 05:24:17,672] {standard_task_runner.py:52} INFO - Started process 8738 to run task
[2023-01-04 05:24:17,678] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-02-24T07:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmpp54j6ygz', '--error-file', '/tmp/tmphr0hy00g']
[2023-01-04 05:24:17,680] {standard_task_runner.py:80} INFO - Job 112: Subtask get_crypto_values
[2023-01-04 05:24:17,743] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-24T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 05:24:17,835] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-02-24T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-24T07:00:00+00:00
[2023-01-04 05:24:18,939] {logging_mixin.py:115} INFO - New rows have been added.
[2023-01-04 05:24:18,939] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 05:24:18,946] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210224T070000, start_date=20230104T082417, end_date=20230104T082418
[2023-01-04 05:24:18,992] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 05:24:19,015] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
